packages/contracts/package.json

```diff
@@ -7,51 +7,46 @@
     "dist/types/*.ts",
     "artifacts/contracts/**/*.json",
     "deployments/**/*.json",
-    "OVM",
-    "iOVM",
-    "libraries",
-    "mockOVM"
+    "chugsplash",
+    "L1",
```

Comment:
Does this need to be `L1/**` or will it recursively work?

Reply:
It works recursively